### PR TITLE
feat(web): Add GitPod support, and run web standalone.

### PR DESCRIPTION
**Pull request by f1shy-dev**
This PR is to allow people to run the web interface connected to the staging API, so people don't need to run the API to make web changes.
 
- [x] Make web run standalone
- [x] Add gitpod support

### CODE REVIEW

**Comment by HenryLeC** [comment]
Looks great

File: package.json [hunk 1]
```diff
@@ -39,5 +39,9 @@
       "path": "./node_modules/cz-conventional-changelog",
       "defaultScope": "global"
     }
+  },
+  "resolutions": {
+    "@oasis/client-gql": "portal:/Users/vrishank/Desktop/OasisDev/oasis-noapi/packages/client-gql",
+    "@oasis/web": "portal:/Users/vrishank/Desktop/OasisDev/oasis-noapi/packages/web"
```
Comment (heybereket) [inline package.json]: Does this need to be under a certain user (you in this case?)

**Review by heybereket — COMMENTED** [review]
Looks good! Just one comment:

File: package.json (hunk 1 above)
Comment (f1shy-dev) [reply package.json]: I think yarn link generated those, I'll fix that ig

**Pull request merged**